azurerm/internal/services/subscription/data_source_subscriptions.go

```diff
@@ -99,6 +104,9 @@ func dataSourceSubscriptionsRead(d *schema.ResourceData, meta interface{}) error
 
 		s := make(map[string]interface{})
 
+		if v := val.ID; v != nil {
+			s["resource_manager_id"] = *v
```

Comment:
can we make this just `id` to match other resources:

```suggestion
			s["id"] = *v
```